syft/core/frameworks/torch/tensor.py

```diff
@@ -364,9 +364,18 @@ def get(self, deregister_ptr=True, update_ptr_wrapper=True):
         # still exist referencing this pointer will simply call local data instead
         # of sending messages elsewhere, or a closer pointer
         if update_ptr_wrapper:
-            # TODO: modifying self is maybe not a good practice, but it's the only way I found
-            # to ensure that x.get() which shows a tensor is in fact of x.dim() = 1 and not 0
             self.child = syft_tensor
```

Comment:
An alternative here would be to also forward __str__ (like all other methods) to the FloatTensor that's actually holding data. This would be more performant than calling self.set_ as well.